eqc/tcp_proto_eqc.erl

```diff
@@ -108,6 +114,13 @@ get_events() ->
             {get_events, bucket(), min(S, E), max(S, E), filters()}
            ])).
 
+get_metrics() ->
+    Opts = [read_repair_opt(), r_opt()],
```

Comment:
this should probably be it's own generator and also test against empty, and partial lists.